app/src/main/java/ch/hikemate/app/ui/auth/CreateAccountScreen.kt

```diff
@@ -114,11 +120,12 @@ fun CreateAccountScreen(
       modifier =
           Modifier.testTag(Screen.CREATE_ACCOUNT)
               .padding(
-                  // Add for the status bar
                   start = 16.dp,
                   end = 16.dp,
-                  top = 40.dp,
-              ),
+              )
+              .safeDrawingPadding()
+              .verticalScroll(scrollState)
+              .imePadding(),
```

Comment:
Question: I observed that this works in practice when running the app, but I'm not yet sure how it works exactly. Wasn't the `android:windowSoftInputMode="adjustResize"` you added to `AndroidManifest.xml` sufficient?

Reply:
According to [Android documentation](https://developer.android.com/develop/ui/compose/layouts/insets#insets-setup): Set android:windowSoftInputMode="adjustResize" in your Activity's AndroidManifest.xml entry. This setting **allows your app to receive the size of the software IME as insets**, which you can use to pad and lay out content appropriately when the IME appears and disappears in your app.

Thus, we receive the padding, but we still need to apply it!

Reply:
Alright, makes no sense, as always with Android. But you made it work somehow anyway

Reply:
Alright, makes no sense to do things this way, as usual with Android, but I get it now.